src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExceptionListenerPass.php

```diff
@@ -36,5 +36,9 @@ public function process(ContainerBuilder $container)
                 $container->removeDefinition('twig.exception_listener');
             }
         }
+
+        if ($container->hasDefinition('twig.exception_listener')) {
+            $container->removeDefinition('http_exception_listener');
```

Comment:
I'd suggest to not conditionally unregister the listener, let it be, that's not a measurable overhead anyway